tests/backup/backup_upgrade_test.go

```diff
@@ -204,3 +206,541 @@ var _ = Describe("{StorkUpgradeWithBackup}", func() {
+			nextScheduleBackupOrdinal := currentScheduleBackupCount + 1
+			checkOrdinalScheduleBackupCreation := func() (interface{}, bool, error) {
+				ordinalScheduleBackupName, err := GetOrdinalScheduleBackupName(ctx, allNSScheduleName, nextScheduleBackupOrdinal, orgID)
+				if err != nil {
+					return "", true, err
+				}
+				return ordinalScheduleBackupName, false, nil
+			}
+			log.InfoD("Waiting for %v minutes for the next schedule backup to be triggered", intervalInMins)
+			time.Sleep(time.Duration(intervalInMins) * time.Minute)
+			nextScheduleBackupName, err := DoRetryWithTimeoutWithGinkgoRecover(checkOrdinalScheduleBackupCreation, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching name of the next schedule backup with schedule [%s]", allNSScheduleName))
+			log.Infof("Next schedule backup name: [%s]", nextScheduleBackupName.(string))
+			err = backupSuccessCheck(nextScheduleBackupName.(string), orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying success of next schedule backup [%s] with schedule [%s]", nextScheduleBackupName.(string), allNSScheduleName))
+			restoreName := fmt.Sprintf("%s-%s", "test-restore-all-ns", RandomString(4))
+			log.InfoD("Restoring next allNS schedule backup [%s] in cluster [%s] with restore [%s]", nextScheduleBackupName.(string), SourceClusterName, restoreName)
+			namespaceMapping := make(map[string]string, 0)
+			err = CreateRestore(restoreName, nextScheduleBackupName.(string), namespaceMapping, SourceClusterName, orgID, ctx, make(map[string]string, 0))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying restoration [%s] of next allNS schedule backup [%s] in cluster [%s]", restoreName, nextScheduleBackupName.(string), restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+	})
+	JustAfterEach(func() {
+		allContexts := append(srcClusterContexts, destClusterContexts...)
+		defer EndPxBackupTorpedoTest(allContexts)
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		log.InfoD("Deleting singleNS Backup schedules %s", singleNSScheduleNames)
+		deleteSingleNSScheduleTask := func(scheduleName string) {
+			scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+			log.FailOnError(err, "Fetching uid of singleNS backup schedule [%s]", scheduleName)
+			log.Infof("SingleNS backup schedule [%s] uid: [%s]", scheduleName, scheduleUid)
+			log.InfoD("Deleting singleNS backup schedule [%s]", scheduleName)
+			err = DeleteSchedule(scheduleName, scheduleUid, orgID)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Verifying deletion of backup schedule [%s]", scheduleName))
+		}
+		_ = TaskHandler(singleNSScheduleNames, deleteSingleNSScheduleTask, Parallel)
+		log.InfoD("Deleting allNS Backup schedule %s", allNSScheduleName)
```

Comment:
Same

Reply:
Changed all "allNS" in string literals to "all namespaces".

Reply:
Changed all "allNS" in comments as well to "all namespaces".